programs/locker/src/state/claim_status.rs

```diff
@@ -0,0 +1,15 @@
+use crate::*;
+
+use safe_math::SafeMath;
+#[account]
+#[derive(Default, InitSpace, Debug)]
+pub struct ClaimStatus {
+    pub total_claimed_amount: u64,
```

Comment:
could you add some padding here, incase we need to store more fields

Reply:
Could you store recipient/ and escrow address in this account? and add a function to close claimStatus to get rent fee if the escrow is closed or cancel?

Reply:
Done.

Reply:
I added new instruction for it.